api/translations/validators.de.yml

```diff
@@ -0,0 +1,5 @@
+Either this value or {{ other }} should not be null.: "Entweder dieser Wert oder {{ other }} müssen nicht null sein"
```

Comment:
Tönt etwas holprig. Vorschlag:
```suggestion
Either this value or {{ other }} should not be null.: "Dieser Wert und {{ other }} dürfen nicht beide null sein."
```

Reply:
I propose we add the translations themselves in a later PR.